x-pack/plugins/upgrade_assistant/public/application/components/kibana_deprecations/deprecation_details_flyout.tsx

```diff
@@ -203,16 +203,26 @@ export const DeprecationDetailsFlyout = ({
               <h3>{i18nTexts.manualFixTitle}</h3>
             </EuiTitle>
 
-            <EuiSpacer />
+            <EuiSpacer size="s" />
```

Comment:
I reduced the size of this spacer, to match the spacing between headings and paragraphs in [EuiText](https://elastic.github.io/eui/#/display/text).